### ISSUE TITLE
Static Push Publishing fails to send one single File Asset

### ISSUE THREAD

**Issue by fabrizzio-dotCMS** [dotCMS/core#22073]
**Describe the bug**
 Found in Our Current Master - seems like it has been around for a while
 When Sending one single File-Asse to an S3 Bucket the Generated bundle is Empty

**To Reproduce**
1. Set up an s3 Bucket endpoint for push-publish 
2. Upload a fresh File Asset
3. Push publish the File asset into the S3 endpoint. (Selecting the Force-Push Option)
4. See on the logs a warning like: 

![Screen Shot 2022-04-25 at 10 06 46](https://user-images.githubusercontent.com/37148755/165131254-1d15ca0b-ecea-4d19-9403-e5fe845710d2.png)

 This behavior is specific to Static Push. Regular Push publish is working as expected.

**Expected behavior**
We should be to force push an asset at any moment.

**Additional context**
I believe the error comes from this class `AWSS3Publisher`
Around line 141/142

The following condition is inverted and should probably set the result to true when the bucket is valid and not otherwise

```
if (!endPointPublisher.exists(bucketName)) {
      result = true;
     break outerLoop;
}
```

**Comment by nollymar** [comment]
PR EE: https://github.com/dotCMS/enterprise/pull/874
PR Core: #22200

**Comment by nollymar** [comment]
**Internal QA:** Passed

**Comment by bryanboza** [comment]
Fixed, tested release 22.07 and I'm able to push individual assets without problems.
